sbnana/CAFAna/Core/SpectrumLoader.cxx

```diff
@@ -112,6 +112,54 @@ namespace ana
     fHistDefs.Clear();
   }
 
+  //----------------------------------------------------------------------
+  void SpectrumLoader::Select(unsigned int N)
+  {
+    if(fGone){
+      std::cerr << "Error: can only call Go() once on a SpectrumLoader" << std::endl;
+      abort();
+    }
+    fGone = true;
```

Comment:
Would you not potentially want to be able to select several events one after the other?